playbooks/roles/genesis-google/tasks/main.yml

```diff
@@ -28,7 +28,7 @@
   add_host:
     name: "{{ streisand_server.instance_data[0].public_ip }}"
     groups: streisand-host
-    ansible_user: ubuntu
+    ansible_user: root
```

Comment:
I think we can just omit the `ansible_user` and `ansible_become` lines entirely as we do for other providers that use `root` (e.g. DigitalOcean). The `streisand.yml` playbook [specifies default values for these variables](https://github.com/jlund/streisand/blob/master/playbooks/streisand.yml#L8) and it looks like we don't need to override them anymore.

My speculative guess is that GCE has moved away from the `ubuntu` user in their images in an attempt to be more consistent, though they've now diverged from Azure and EC2 who still don't use `root` for new installs. I will double-check to make sure I am seeing this behavior in my GCE account as well.